pkg/ddc/alluxio/node_test.go

```diff
@@ -169,6 +177,10 @@ func TestSyncScheduleInfoToCacheNodes(t *testing.T) {
 				name:      "hbase-a",
 				namespace: "big-data",
 				worker: &appsv1.StatefulSet{
+					TypeMeta: metav1.TypeMeta{
+						Kind:       "StatefulSet",
+						APIVersion: "apps/v1",
+					},
```

Comment:
Why is this necessary in new version of controller-runtime?

Reply:
It seems related to https://github.com/kubernetes-sigs/controller-runtime/pull/2633. If removed, test cases will fail.